src/components/dropdown/data.tsx

```diff
@@ -61,69 +61,79 @@ export const prepareProposalsData: DropdownTopic[] = [
   },
 ]
 
-const checkSteps = async (items: DropdownItem[], balances: TokenBalanceRecord, address: Address) => {
-  const balanceDependentSteps = checkBalancesSteps(items, balances)
-  try {
-    const checkedEvents = await checkEvents(balanceDependentSteps, address)
-
-    return checkedEvents
-  } catch (err) {
-    console.log('CHECKING THE STEPS FAILED', err)
-    return balanceDependentSteps
-  }
-}
-
-const checkBalancesSteps = (items: DropdownItem[], balances: TokenBalanceRecord): DropdownTopic[] => {
+const checkBalancesSteps = (items: DropdownItem[], balances: TokenBalanceRecord) => {
```

Comment:
Can we have some sort of jsdoc here briefly explaining what is the objective of this function?